lib/src/parse/stylesheet.dart

```diff
@@ -1403,6 +1403,12 @@ abstract class StylesheetParser extends Parser {
         break;
 
       case $0:
+        if (scanner.peekChar(1) == $backslash && scanner.peekChar(2) == $0) {
```

Comment:
Please leave fixes for other issues outside of this PR. One thing at a time makes it easier to review.